src/UnitTests/Projection/MapFromTest.cs

```diff
@@ -21,6 +21,18 @@ public void Should_not_fail()
         }
 
         [Fact]
+        public void Should_not_fail_Untyped()
+        {
+            var config = new MapperConfiguration(cfg =>
+            {
+                cfg.CreateMap<UserModel, UserDto>()
+                    .ForMember(dto => dto.FullName, opt => opt.MapFrom(src => src.LastName + " " + src.FirstName));
+            });
+
+            typeof(ArgumentNullException).ShouldNotBeThrownBy(() => config.ExpressionBuilder.GetMapExpression(typeof(UserModel), typeof(UserDto), null, null)); //no ArgumentNullException here
```

Comment:
But the Typed API `` Expression<Func<TSource, TDestination>> GetMapExpression<TSource, TDestination>(ParameterBag parameters = null, params MemberInfo[] membersToExpand);`` has a ``params`` for the membersToExpand and the untyped not. the ``params`` will lead to an empty array automatically, even if no ``MemberInfo``s are passed. I just fixed the difference in behaviour when I pass no ``MemberInfo``s. It's just consistent to handle it exactly like your ``parameters = parameters ?? new Dictionary<string, object>();``